### DISCUSSION TITLE
Notification feature with custom thresholds

### DISCUSSION THREAD

**Discussion by Discostu36** [electricitymaps/electricitymaps-contrib#3004] [Ideas]
I'd love it if the app had a notification feature. Something like "renewable energy in Germany is above 60 % threshold!" Then I could load my pedelec, smartphone etc. at that moment to lower my carbon footprint.

**Comment by madsnedergaard** [comment]
Hey @Discostu36, we have created an electricityMap service on [ifttt.com/electricitymap](https://ifttt.com/electricitymap).

Maybe you can use that for now? :)

<img width="615" alt="Screenshot 2021-02-08 at 16 03 17" src="https://user-images.githubusercontent.com/3296643/107237605-35929600-6a27-11eb-9d76-0b01204e4326.png">

**Comment by Discostu36** [comment]
This is a good start, thanks. Only thing that is missing is setting a custom threshold. I'm in Germany, so if I only get notified as soon as Germany is below 150g/kwh, it will not happen very often.

**Comment by Kongkille** [comment]
I'll move this to discussions. I think IFTTT can solve a lot of these issues but I understand that it doesn't show custom thresholds

**Comment by bboulesteix** [comment]
Thanks for opening this discussion. I'd love to be able to set thresholds in the IFTTT app too, but for the time being I can't even receive a proper notification from the IFTTT app when electricity is low/high carbon in France. So here's my question: what triggers the following two IFTTT triggers?

1. https://ifttt.com/electricitymap/triggers/high_carbon_by_zone_key
2. https://ifttt.com/electricitymap/triggers/low_carbon_by_zone_key

The [original blogpost](https://www.electricitymaps.com/blog/ifttt-blogpost) from Electricty Map's blog says:

> By using [our new Electricity Maps service on IFTTT](https://ifttt.com/electricitymap), it is possible for you or your smart devices to automatically get a notification **when local electricity has the lowest carbon intensity during a 24 hour period**.

And here's what `@corradio` [replied on Twitter](https://twitter.com/corradio/status/1280011040181432321) when asked what was considered lowest/highest carbon intensity:

> It's the highest/lowest point in our forecast for the next 24hrs

On my IFTTT account, I set up the two following applets:

1. If electricity is **high** carbon in France, then send a notification from the IFTTT app.
3. If electricity is **low** carbon in France, then send a notification from the IFTTT app.

Now, if I read your blogpost well, I should receive 2 notifications from the IFTTT app on a given day: one at the lowest carbon intensity point and the other one at the highest point—regardless of the carbon intensity of France relative to other countries. And yet, IFTTT tells me that my applets were _never_ run (both of them). In other words: I never got any notification from my ElectricityMap applets.

Am I doing something wrong? Are notifications never sent in France because French electricity is low carbon most of the time? According to IFTTT, your [API seems to be working just fine](https://ifttt.com/electricitymap/health), so I assume that this is not an endpoint issue.
